internal/api/cluster.go

```diff
@@ -575,5 +548,5 @@ func handleGetAllUtilityMetadata(c *Context, w http.ResponseWriter, r *http.Requ
 
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusOK)
-	w.Write(cluster.UtilityMetadata)
+	outputJSON(c, w, cluster.UtilityMetadata)
```

Comment:
Can we change this to `writeJSON` so that it's consistent with `WriteHeader`? 0/5

Reply:
Sounds good. I just reused the the existing function, but I will update the name on my next pass.